### PR TITLE
Feature/wmt 470/extract parser

### PR DESCRIPTION
**Pull request by tswann**
Functionality to parse the WMT extract file
- openpyxl/pandas parser to validate and load workbook
- Base package structure and unit tests
- Travis CI integration

### CODE REVIEW

File: wmt_etl/extract_parser.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    return [clean_name(name) for name in names]
+
+def clean_name(name):
+    ''' Rules for transforming an extract worksheet or column name to staging table format'''
+    illegal_chars = '-'
+    return name.strip().replace(illegal_chars, '').lower()
+
+def validate_workbook_format(sheet_names):
+    '''XLSX file should be in the expected format'''
+    return set(transform_names(sheet_names)) == set(config.VALID_SHEET_NAMES)
```
Comment (andrew-js-wright) [inline wmt_etl/extract_parser.py]: Do we require that the sheets included in the workbook are the exact list specified in the ETL config or is it enough to ensure that `VALID_SHEET_NAMES` is a sub set of `workbook.sheet_names`?

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+`workon wmt-etl`
+
+And it will switch to this virtualenv and take you to the ./wmt-etl directory.
+
+### Running the ETL Job
+
+To run the example ETL file, first ensure that the local development DB is running and execute the following script to submit it for execution:
+
+`python etl.py`
```
Comment (andrew-js-wright) [inline README.md]: This is now `python run.py`

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+`mkvirtualenv -a wmt-etl -r requirements.txt wmt-etl` 
+
+All subsequent times you want to work on this project you should just need to type:
+
+`workon wmt-etl`
+
+And it will switch to this virtualenv and take you to the ./wmt-etl directory.
+
+### Running the ETL Job
+
```
Comment (andrew-js-wright) [inline README.md]: Might be worth signposting to the worker repo to flag the dependency on the database being there before ETL will work.

**Review by scoch — APPROVED** [review]
LGTM 👍 

For invalid filename patterns, this will never change will it? Where for e.g. we could get something like:

 my example_work _-book-invalid

**Review by andrew-js-wright — APPROVED** [review]

File: wmt_etl/extract_parser.py (hunk 1 above)
Comment (tswann) [reply wmt_etl/extract_parser.py]: A subset would be enough yes - there's no need to be overly strict in that sense. I'll update.

File: README.md (hunk 3 above)
Comment (tswann) [reply README.md]: 👍

**Comment by tswann** [comment]
Updates applied.

**Pull request merged**